### ISSUE TITLE
Possible New Feature: Identifiers Tags for cards

### ISSUE THREAD

**Issue by stripathi669** [dexafree/MaterialList#45]
In a scenario where cards are being used a templates for large amount of new content, it is important to have some identifier ID for each card instance apart from the just index position.

Though this small feature is relatively simple to add, but it has great utility.

**Comment by dexafree** [comment]
You mean by

```java
Card c = new Card(mContext, "JOHN");
mList.add(c);

// do some stuff

Card johnsCard = mList.get("JOHN");
```

**Comment by stripathi669** [comment]
I think I have not been able to express the thought properly.

Say the cards are displaying some unit of content, and each unit has unique identifier. This unique identifier is useful as it identifies what content was displayed on the card in callbacks.

Your code is about how to identify a card using a Tag, however I am talking about identifying content that will be displayed.

PS: I don't understand how the code you gave is supposed to work, since card is abstract class and can't be instantiated.

**Comment by dexafree** [comment]
Using the `Card` class was just made as an example.

I think I understand what you are saying.

Which class are you thinking to be used as Tag? Object and let every developer use their own tags? Strings for all?

**Comment by stripathi669** [comment]
Yes, I understand.

Now I think it is best to let every developer use their own tags. Probably each developer can add their own system for this, so don't worry about it. Your time is better spent on other cool features that might have in line.

**Comment by dexafree** [comment]
I'm a student, and believe me when I say that I am learning a lot while developing this library!
I will take a look at it this week ;)

**Comment by dexafree** [comment]
Already available on version 2.4.0 :+1:
